crates/runtime/src/lib.rs

```diff
@@ -20,23 +20,19 @@ pub use runtime::create_rocksdb_runtime;
 mod gas;
 pub use gas::DefaultGasEstimator;
 
-/// Managing the Runtime's environment
+/// Managing the Runtime environment
 mod env;
 pub use env::{Env, EnvTypes};
 
-mod import;
 mod storage;
 
-pub use import::ExternImport;
-
 /// Implements `Context`. Used for managing data of running `SVM` apps.
 mod context;
 pub use context::Context;
 
-/// Implements common functionality to be consnumed by tests.
+/// Implements common functionality to be consumed by tests.
```

Comment:
I think it's best to use inner doc comments (`//!`) for `mod foobar;` rather than outer doc comments (`///`). Basically the same way we have for `vmcalls/mod.rs`.

https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text